### PR TITLE
feat: Add BoundSwitchField, FormLines.

### PR DESCRIPTION
**Pull request by stephenh**
For Trade Partner Contact screens in internal-internal.

FormLines is kinda WIP, but figure it might as well be WIP here instead of copy/pasting it across internal-frontend and procurement-frontend.

### CODE REVIEW

File: src/inputs/Switch.tsx [hunk 1]
```diff
@@ -13,6 +14,8 @@ export interface SwitchProps {
   disabled?: boolean;
   /** Input label */
   label: string;
+  /** Where to put the label. */
+  labelStyle?: "form" | "inline";
```
Comment (stephenh) [inline src/inputs/Switch.tsx:18]: The current switch had labels off to the right, but when in a form with other fields, I think it's more consistent to put the label on top.

File: src/inputs/Switch.tsx [hunk 2]
```diff
@@ -42,27 +46,30 @@ export function Switch(props: SwitchProps) {
     <label
       {...hoverProps}
       css={{
-        ...switchLabelDefaultStyles,
```
Comment (stephenh) [inline src/inputs/Switch.tsx:45]: I inlined a bunch of these styles that were constants b/c the idea of tachyons/utility css is to have the css defined inline / as close to the markup as possible.

File: src/forms/FormLines.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        "& > *": Css.mb3.$,
+      }}
+    >
+      {children}
+    </div>
+  );
+}
+
+/** Groups multiple fields side-by-side. */
+export function FieldGroup(props: {
```
Comment (KoltonG) [inline src/forms/FormLines.tsx:31]: **Question:** Is this like FormColumn?

File: src/forms/FormLines.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      <TextField label={"First"} value={"first"} onChange={() => {}} />
+      <TextField label={"Last"} value={"last"} onChange={() => {}} />
+    </FormLines>
+  );
+}
+
+export function SideBySide() {
+  return (
+    <FormLines>
+      <FieldGroup>
```
Comment (KoltonG) [inline src/forms/FormLines.stories.tsx:31]: **Thoughts**: Any thought on making it `FieldLines` of `formGroup` to keep the names consistent?

**Review by KoltonG — APPROVED** [review]
🍣

File: src/forms/FormLines.tsx (hunk 3 above)
Comment (stephenh) [reply src/forms/FormLines.tsx:31]: Well, it's more like "a multi-field line", i.e.:

```
| first name          | // 1st line
| last name          | // 2nd line
| middle | phone  | // a FieldGroup one the 3rd line
```

**Pull request merged**